src/style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css?family=Karla:weight@400;700&display=swap");
+
+$font: "Karla", sans-serif;
+
+body {
+  font-family: $font;
+}
+
+.main {
+  padding: 10px;
+  border: 1px solid rgb(229, 226, 226);
+  text-shadow: 2px 2px 4px #038b90;
+}
+
+h1 {
+  text-align: center;
+  font-size: 15px;
+}
+
+.todo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.refresh {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+  transition: transform 0.5s;
+}
+
+.icon {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+  float: right;
+}
+
+.refreshed {
+  transform: rotate(360deg);
+}
+
+.list-item {
+  display: flex;
+  padding: 10px;
+  border: 1px solid rgb(229, 226, 226);
+
+  /* box-shadow: 1px 1px 5px rgb(170, 174, 174); */
+  justify-content: space-between;
+}
+
+.yellwo {
+  background-color: rgb(236, 236, 111);
+}
+
+.kebabIcon {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.check-box {
+  float: left;
+}
+
+#input-todo {
+  /* width: auto; */
+  height: 30px;
+  padding-left: 15px;
+  font-style: italic;
+  border: none;
+}
+
+#input-todo:focus {
+  outline: none;
+}
```

Comment:
@saikatgpae, I have noticed the use of an ID selector repeatedly. I would suggest you change this Id selector to a class selector to make your code follow the DRY principle.

Reply:
Thank you for your suggestion.
i removed the unnecessary commented code.
But I used #input-todo:focus for the same id.
I did not repeat actually.